modules/javafx.web/src/main/native/Source/ThirdParty/icu/LICENSE

```diff
@@ -1,6 +1,6 @@
 COPYRIGHT AND PERMISSION NOTICE
 
-Copyright © 1991-2022 Unicode, Inc. All rights reserved.
+Copyright © 1991-2023 Unicode, Inc. All rights reserved.
```

Comment:
This LICENSE file is missing the same preamble you recently added to `icu_web.md` lines 6-18. Or you can copy from:

https://github.com/unicode-org/icu/blob/release-73-1/icu4c/LICENSE

Reply:
Done